AbletonMCP_Remote_Script/handlers/tracks.py

```
@@ -0,0 +1,667 @@
+    try:
+        track = get_track(song, track_index)
+        if target_index is not None:
+            track.insert_device(str(device_name), int(target_index))
+        else:
+            track.insert_device(str(device_name))
+        return {
+            "inserted": True,
+            "device_name": device_name,
+            "track_index": track_index,
+            "track_name": track.name,
+        }
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)